pages/glossary/rendered-image-resource.md

```diff
@@ -13,4 +13,4 @@ A _rendered image resource_ in a [web page][] is any resource with [visible pixe
 [embedded image]: #embedded-image 'Definition of Embedded Image'
 [image sources]: https://html.spec.whatwg.org/multipage/images.html#image-source
 [visible pixels]: #visible 'Definition of visible'
-[web page]: #web-page-html 'Definition of web page (HTML)'
\ No newline at end of file
+[web page]: #web-page-html 'Definition of web page (HTML)'
```

Comment:
As above. Please undo this. Don't want an autocomplete fix to be in the file history.

Reply:
Sorry, not sure how that's happened.

Reply:
@WilcoFiers I managed to get rid of the change in embedded-image file but I'm not sure what is happening here. There is literally no change at all. Do you have any suggestions?

Reply:
I managed to reverse the updating with origin commit which has caused an issue but there was too much fiddling around with it so decided it would be clearer/ easier to cancel the pr and start from the beginning.